Vizzuality/marxan-cloud
# Feature/prettier config

## Prettier config update

### Overview

This PR adds:
* new prettier pre-commit hook (through lint-staged)
* editorconfig

### Testing instructions
Remove end of line space in any file and check the warning. Try to commit it and... fail
![image](https://user-images.githubusercontent.com/6906348/100871402-870b2b80-34a0-11eb-9479-42a4e3951ce7.png)
